src/content/developers/tutorials/how-to-mint-an-nft.md

```diff
@@ -0,0 +1,275 @@
+
+For the more visual learners, the steps above are summarized here:
+
+![How to upload your image to Pinata](https://cdn-images-1.medium.com/max/2000/0*V_-3PGxnfsnLTGo4.gif)
+
+Now, we’re going to want to upload one more document to Pinata. But before we do that, we need to create it!
+
+In your root directory, make a new file called nft-metadata.json and add the following json code:
+
+    {
+        "attributes" : [ {
+          "trait_type" : "Breed",
+          "value" : "Maltipoo"
+        }, {
+          "trait_type" : "Eye color",
+          "value" : "Mocha"
+        } ],
+        "description" : "The world's most adorable and sensitive pup.",
+        "image" : "https://gateway.pinata.cloud/ipfs/QmWmvTJmJU3pozR9ZHFmQC2DNDwi2XJtf3QGyYiiagFSWb",
+        "name" : "Ramses"
+    }
+
+Feel free to change the data in the json. You can remove or add to the attributes section. Most importantly, make sure image field points to the location of your IPFS image — otherwise, your NFT will include a photo of a (very cute!) dog.
+
+Once you’re done editing the json file, save it and upload it to Pinata, following the same steps we did for uploading the image.
+
+![How to upload your nft-metadata.json to Pinata](https://cdn-images-1.medium.com/max/2000/0*eFbjFolSc50YMBmO.gif)
+
+## Step 5: Create an instance of your contract
+
+Now, to interact with our contract, we need to create an instance of it in our code. To do so we’ll need our contract address which we can get from the deployment or [Etherscan](https://ropsten.etherscan.io/) by looking up the address you used to deploy the contract.
+
+![View your contract address on Etherscan](https://cdn-images-1.medium.com/max/5200/0*kLL79TiiBrOHjBQR.png)
+
+In the above example, our contract address is 0x81c587EB0fE773404c42c1d2666b5f557C470eED.
+
+Next we will use the web3 [contract method](https://web3js.readthedocs.io/en/v1.2.0/web3-eth-contract.html?highlight=constructor#web3-eth-contract) to create our contract using the ABI and address. In your mint-nft.js file, add the following:
+
+    const contractAddress = "0x81c587EB0fE773404c42c1d2666b5f557C470eED";
+
+    const nftContract = new web3.eth.Contract(contract.abi, contractAddress);
+
+## Step 6: Update the .env file
+
+Now, in order to create and send transactions to the Ethereum chain, we’ll use your public ethereum account address to get the account nonce (will explain below).
+
+Add your public key to your .env file —if you completed part 1 of the tutorial, our .env file should now look like this:
+
+    API_URL = "https://eth-ropsten.alchemyapi.io/v2/your-api-key"
+    PRIVATE_KEY = "your-private-account-address"
+    PUBLIC_KEY = "your-public-account-address"
+
+## Step 7: Create your transaction
+
+First, let’s define a function called mintNFT(tokenData) and create our transaction by doing the following:
+
+1. Grab your PRIVATE_KEY* *and* *PUBLIC_KEY from the .env file.
```

Comment:
This renders
![image](https://user-images.githubusercontent.com/54227730/113186055-066cf980-920c-11eb-8874-442aee84d676.png)
Was that intentional?
Typically we're using single underscore wrapping to denote italics, and double asterisk wrapping to denote bold, which must touch the content they're wrapping without a space.